src/components/DisbursementInviteMessage/styles.scss

```diff
@@ -0,0 +1,30 @@
+@use "./styles-utils.scss" as *;
```

Comment:
This file is exactly the same as `src/components/ReceiverInviteMessage/styles.scss`. We should remove it. It looks like the `DisbursementInviteMessage` component is using the same styles.